test/net/sourceforge/kolmafia/request/BasementRequestTest.java

```diff
@@ -19,19 +17,16 @@ protected static void injectPreferences() {
     KoLCharacter.reset("fakeUserName");
   }
 
-  private static Stream<Arguments> monsterFights() {
-    return Stream.of(
-        Arguments.of("Commence to Pokin'", "Beast with X Eyes"),
-        Arguments.of("Don't Fear the Ear", "Beast with X Ears"),
-        Arguments.of("Stone Golem'", "X Stone Golem"),
-        Arguments.of("Hydra'", "X-headed Hydra"),
-        Arguments.of("Toast that Ghost'", "Ghost of Fernswarthy's Grandfather"),
-        Arguments.of("Bottles of Beer on a Golem", "X Bottles of Beer on a Golem"),
-        Arguments.of("Collapse That Waveform!", "X-dimensional horror"));
-  }
-
   @ParameterizedTest
-  @MethodSource("monsterFights")
+  @CsvSource({
```

Comment:
while personal preference, I prefer the external strongly-typed argument to the stringly-typed csv source ;)

Reply:
I think this just lends itself to really easy test writing, and it's surprisingly hard to make mistakes with it